### ISSUE TITLE
Show file names in thumbnail overview

### ISSUE THREAD

**Issue by tschoeni** [biigle/core#129]
People need to know image filenames. Humans think with names and when searching for images they would prefer to also have the filename as a key. I've had many users ask why the filenames are not there anymore. I know its adding a lot of text to the interface which might not be deemed beautiful but it speeds up finding an image. I know there is the filter / search option but whithin one thumbnail "page" you are much faster finding an image by name yourself rather than having to click four times and tiping the name (which you will probably not remember exactly anyway).

TL;DR: Please add the filenames to thumbnail overview.
